runtime/compiler/runtime/GPUHelpers.cpp

```diff
@@ -1887,41 +1887,50 @@ bool freeGPUScope(CudaInfo *cudaInfo)
 
    if (cudaInfo)
       {
-         if (cudaInfo->hashTable)
-            {
-            TR_Memory::jitPersistentFree(cudaInfo->hashTable);
-            }
-
-         TR_Memory::jitPersistentFree(cudaInfo);
+      if (cudaInfo->hashTable)
+         {
+         TR_Memory::jitPersistentFree(cudaInfo->hashTable);
+         }
+      TR_Memory::jitPersistentFree(cudaInfo);
       }
 
    return failedGPUSession;
    }
 
 extern "C" int
-regionExitGPU(CudaInfo *cudaInfo, uint8_t *startPC, int ptxSourceID, int flushBlock, void **liveRef)
+regionExitGPU(CudaInfo *cudaInfo, uint8_t *startPC, int ptxSourceID, void **liveRef)
    {
-   returnOnTrue(!cudaInfo, NULL_DEVICE_PTR);
+   returnOnTrue(!cudaInfo, 1);
```

Comment:
Here we return 1 while we return 0 in other places. We definitely should not use a constant but some macro. I think we have some to indicate error/ok condition.

Reply:
Would it be better to have it return 0 instead of 1 on an error? Right now nothing looks at the return value. I changed this line to return 1 on error but it is also possible to do the reverse and change this line:
```
return (failedGPUSession ? 1 : 0);
``` 
to return 0 on error instead.

Reply:
It's exactly my point:  I am not sure what 0 or 1 means in each situation. I think we need to return some macro or an enum, similar to some similar calls in this file.

Reply:
Since @IBMJimmyk will open another PR for cleaning up return codes I am closing this.

Reply:
I'm going to leave this as just a magic number of 1 for now since unclear is better than NULL_DEVICE_PTR which was wrong. A more detailed clean up of error codes is outside the scope of this PR. I will shortly be submitting a new PR with the more detailed error code clean up.